docusaurus/docs/reference/deployments/01-controller.md

```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
+## Bootstrapping
+
+Bootstrapping happens automatically unless you disable it before running the controller service. Bootstrapping a controller involves:
+
+1. Generating a PKI
+1. Generating a configuration YAML file
+1. Initializing a database with a default admin password
+
+At a minimum, you must set these to start the service with bootstrapping enabled (the default).
+
+1. In `/opt/openziti/etc/controller/env` (the systemd env file for the controller service)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller
+    1. Temporarily set `ZITI_PWD` to the desired management password for user `admin`
+
+### Bootstrapping the Public Key Infrastructure
+
+Check out [the pki page](/learn/core-concepts/pki.md) for an overview of the concepts used in this deployment reference. The controller service will bootstrap a root and intermediate CA during first startup, unless they exist. 
+
+The controller needs a CA to issue certificates for edge identities during enrollment. This CA is known as the edge signer. It's a good idea to use an intermediate CA for this purpose, as it allows you to rotate the edge signer without changing the root of trust and invalidating all edge enrollments.
+
+The controller needs at least one client and server certificate. The client certificate is used to authenticate to other controllers in a multi-controller deployment.
+
+Disable bootstrapping a PKI by setting `ZITI_BOOTSTRAP_PKI=false` in `/lib/systemd/system/ziti-controller.service`.
+
+### Bootstrapping the Configuration
+
+The controller's configuration is loaded from a YAML file ([reference](/reference/30-configuration/controller.md)). The Linux system service will generate a valid configuration during the first startup, unless one already exists. 
+
+The filename of the configuration file, relative to the Linux service's working directory (`/var/lib/ziti-controller`), is specified in `/lib/systemd/system/ziti-controller.service` as an argument to the `ExecStart` command.
+
+Disable bootstrapping a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/lib/systemd/system/ziti-controller.service`.
+
+### Bootstrapping the Database
+
+The controller requires a BoltDB database to store its state. The Linux system service will initialize a database with a default admin password during the first startup, unless one already exists.
+
+You must specify the management password for the default admin user before starting the service. This is done by setting `ZITI_PWD` in `/opt/openziti/etc/controller/env` or one of `LoadCredential` or `SetCredential` in `/lib/systemd/system/ziti-controller.service`. You may delete the password after bootstrapping for security.
+
+Disable bootstrapping the database by setting `ZITI_BOOTSTRAP_DATABASE=false` in `/lib/systemd/system/ziti-controller.service`.
+
+## Bring Your Own State

```

Comment:
let's extract this to it's own page and lets focus on not making this a generic example, but focus specifically on "migrating a quickstart"